app/src/main/java/com/android/bookswap/data/DataPhoto.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.bookswap.data
+
+import android.net.Uri
+
+data class DataPhoto(
+    val id: String,
+    val url: String,
+    val timestamp: Long,
+    val userId: String,
+    val bookId: String,
```

Comment:
We need to discuss whether the bookId is necessary, but it should be fine for the moment, as a picture doesn't necessarily have to be related to an image, but it could always be null